concent_api/core/tests/test_unit_transfer_operation.py

```diff
@@ -106,8 +106,8 @@ def test_that_file_transfer_token_for_concent_is_never_out_of_date(self):
         token = create_file_transfer_token_for_concent(
             subtask_id=report_computed_task.subtask_id,
             source_package_path=get_storage_source_file_path(report_computed_task.task_id, report_computed_task.subtask_id),
-            source_size=report_computed_task.task_to_compute.size,
-            source_package_hash=report_computed_task.task_to_compute.package_hash,
+            source_size=report_computed_task.task_to_compute.size,  # pylint: disable=no-member
+            source_package_hash=report_computed_task.task_to_compute.package_hash,  # pylint: disable=no-member
```

Comment:
Please upgrade your pylint requirements and delete this disables because in new version it isn't warning :)

Reply:
In which version ? I have pylint v1.9.1, and the warning is being show.

Reply:
done